docs/import-data.md

```diff
@@ -130,6 +131,39 @@ This is a util that can help generate the import json data needed by the import
 4. Use the generated data to feed into [import] command (/import.md) to generate kick off the import.
 
 
+## Bitbucket Cloud
+*Please note that this tool uses Bitbucket cloud API version 2.0*
+1. Set the Username and Password credentials for your Bitbucket Cloud as environment variables:
```

Comment:
`Username and Password` => `username and password`

Reply:
Done